src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareFunction = (a, b) => {
+    const strA = String(a);
+    const strB = String(b);
+
+    if (strA > strB) {
+      return 1;
+    } else {
```

Comment:
If you use return inside `if` block, writing `else` is useful. This is the correct structure:
```
if (condition1) {
  return x;
};

if (condition2) {
  return y;
};

return z;
```